server/seed/index.js

```diff
@@ -1,14 +1,22 @@
 var faker = require('faker');
-const models = require('./models/index.js');
-//usernames
-var randomUsername = faker.internet.userName();
-// console.log('randomUsername', randomUsername);//Elton_Hauck
-//product names
-var randomProductName = faker.commerce.productName();
-// console.log('randomProductName', randomProductName);//Practical Soft Towels
-//url images of those product names
-var randomImageUrl = faker.image.fashion();
-// console.log('randomImageUrl', randomImageUrl);//http://lorempixel.com/640/480/fashion <=this is depression
-
-
-var productArray = [];
+const models = require('../models/index.js');
+
+let productSeed = () => {
+  for (let i = 0; i < 10; i++) {
+    var randomUsername = faker.internet.userName();
+
+    var randomProductItem = faker.commerce.productName();
+
+    var randomArray = [faker.image.fashion(), faker.image.cats(), faker.image.animals()];
+    var randomPic = Math.floor(Math.random() * 3);
```

Comment:
The name of this variable is misleading -- looks more like `randomNumberUpToThree`? You could re-write randomNumber to take in a limit as a parameter, and re-use it whenever you need a random num/index